CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.2.0 (2019-08-14)
+
+- Added support for anchor elements in the dropdown menu component. [#28](https://github.com/blackbaud/skyux-popovers/pull/28)
```

Comment:
Should it be "dropdown component" or are we specifically referring to anchors within the dropdown menu? 

If it's the latter, let's keep it. I ask because its component page just says "dropdown component"

Reply:
Yeah, we have a dropdown menu component that accepts custom buttons :)